### PR TITLE
Feature/add it and pt

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
We don't currently support Italian and Portuguese

# Solution
Add Italian and Portuguese locales

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [x] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/locales/de_DE/de_DE.json [hunk 1]
```diff
@@ -343,7 +343,7 @@
         "title_selfie": "Selfie"
     },
     "poa_guidance": {
-        "button_primary": "Fortfahren",
+        "button_primary": "Continue",
```
Comment (ethan605) [inline src/locales/de_DE/de_DE.json:346]: Is this change intentional?

File: src/locales/de_DE/de_DE.json [hunk 2]
```diff
@@ -447,6 +447,7 @@
         "body_record": "Drücken Sie die Taste, wenn Sie bereit sind",
         "button_primary_finish": "Aufnahme beenden",
         "button_primary_next": "Nächster Schritt",
+        "button_primary_start": "Aufnahme starten",
```
Comment (ethan605) [inline src/locales/de_DE/de_DE.json:450]: Should this be added to `MIGRATION.md`?

**Review by ethan605 — APPROVED** [review]
LGTM, do we need to update the changed keys in `MIGRATION.md`?

File: src/locales/de_DE/de_DE.json (hunk 1 above)
Comment (stefaniacardenas) [reply src/locales/de_DE/de_DE.json:346]: Yes, PoA does not support english at the moment, so for consistency the strings that were translated by mistake were reverted to English

File: src/locales/de_DE/de_DE.json (hunk 2 above)
Comment (stefaniacardenas) [reply src/locales/de_DE/de_DE.json:450]: It's already in the migration file (not sure why the key was missing from some of the locales files)

**Review by kopijunkie — APPROVED** [review]

File: src/locales/de_DE/de_DE.json (hunk 2 above)
Comment (kopijunkie) [reply src/locales/de_DE/de_DE.json:450]: I think this was missing because the "Start recording" button is still the icon button in the SDK.

File: src/locales/de_DE/de_DE.json [hunk 3]
```diff
@@ -270,7 +270,7 @@
         "info_qr_how": "So scannen Sie einen QR-Code",
         "info_qr_how_list_item_camera": "Richten Sie die Kamera Ihres Telefons auf den QR-Code",
         "info_qr_how_list_item_download": "Wenn es nicht funktioniert, laden Sie einen QR-Code-Scanner von Google Play oder aus dem App Store herunter",
-        "link_divider": "oder",
+        "link_divider": "oder wählen Sie eine dieser Alternativen",
```
[the remark this answers is not in the document]
Comment (stefaniacardenas) [reply src/locales/de_DE/de_DE.json:273]: I left those strings in otherwise I would have had to change the Italian and Portuguese translations
